lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,22 @@
-
+require 'base64'
+require 'securerandom'
```

Comment:
True, not anymore - that method got moved around and this was forgotten there. I'll remove it.